src/Packetery/Module/Helper.php

```diff
@@ -19,6 +19,49 @@
  */
 class Helper {
 
+	/**
+	 * Callback for array filter. Returns true if gateway is of correct type.
+	 *
+	 * @param object $gateway Gateway to check.
+	 *
+	 * @return bool
+	 */
+	protected static function filterValidGatewayClass( object $gateway ): bool {
+		return $gateway instanceof WC_Payment_Gateway;
+	}
+
+	/**
+	 * Get available gateways.
+	 *
+	 * @return WC_Payment_Gateway[]
+	 */
+	public static function getAvailablePaymentGateways(): array {
```

Comment:
Nebylo by lepší udělat na tyto tři metody službu? Nepřijde mi to moc jako úloha pro obecný Helper.